src/Widgets/index.ts

```diff
@@ -2,6 +2,7 @@ import { initNeoletterFormWidgets } from 'scrivito-neoletter-form-widgets'
 import.meta.glob(['./**/*WidgetClass.ts', './**/*WidgetComponent.tsx'], {
   eager: true,
 })
+import './migrateAttributeNameToData'
```

Comment:
* [ ] Why make this part of the code history? It looks like a one-shot migration that can be run on the console.

Reply:
It is a one-shot migration. But not a trivial one, that is why I wanted to have it at least in a file, that I can later restore if needed.

Reply:
How about a branch in a fork?
Making the script part of the history has some drawbacks:
* It creates noise in the history
* There are no guidelines for a reviewer for how to review code and/or functionality

Reply:
I forced pushed a version without those two commits.